File: starter-bots/KotlinBot/src/main/kotlin/arctica/service/model/dto/BotDto.kt
import arctica.service.model.Position
import arctica.service.model.BotMapState
import arctica.service.model.dto.PlayerActionDto
import java.util.*

/**
 *The player's local map
 */

data class BotDto(
    val id: UUID,
    val currentTierLevel: Int,
    val tick: Int,
    val map: BotMapState,
    val population: Int,
    val baseLocation: Position,
    val PendingActions: List<PlayerActionDto>,
    val Actions: List<PlayerActionDto>,
/*    val farmingUnits: Int,
    val lumberingUnits: Int,
    val miningUnits: Int,
    val farmingUnits: Int,
    val farmingUnits: Int,*/
    val availableUnits: Int,
    val seed: Int,
    val wood: Int,
    val food: Int,
    val stone: Int,
    val heat: Int
)

File: game-runner/Domain/Models/DTOs/BotDto.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;
using Newtonsoft.Json;

// The player's local map

namespace Domain.Models.DTOs
{
    public class BotDto
    {
        public Guid Id { get; set; }
        public int CurrentTierLevel { get; set; }
        public int Tick { get; set; }
        public BotMapState Map { get; set; }
        public int Population { get; set; }
        public Position BaseLocation { get; set; }
        public List<PlayerActionDto> PendingActions { get; set; }
        public List<PlayerActionDto> Actions { get; set; }
        // public int TravellingUnits { get; set; }
        // public int LumberingUnits { get; set; }
        // public int MiningUnits { get; set; }
        // public int FarmingUnits { get; set; }
        // public int ScoutingUnits { get; set; }
        public int AvailableUnits { get; set; }
        public int Seed { get; set; }
        public int Wood { get; set; }
        public int Food { get; set; }
        public int Stone { get; set; }
        public int Heat { get; set; }
    }
}